Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,628 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
+| South America | Mayo-kebbi                |
+| South America | Ouaddai                   |
+| South America | Tandjile                  |
+| South America | Ane                       |
+| South America | Kabyé                     |
+| South America | Kotokoli                  |
+| South America | Moba                      |
+| South America | Naudemba                  |
+| South America | Watyi                     |
+| South America | Kuy                       |
+| South America | Tokelau                   |
+| South America | Arabic-French             |
+| South America | Arabic-French-English     |
+| South America | Ami                       |
+| South America | Atayal                    |
+| South America | Min                       |
+| South America | Paiwan                    |
+| South America | Chaga and Pare            |
+| South America | Gogo                      |
+| South America | Ha                        |
+| South America | Haya                      |
+| South America | Hehet                     |
+| South America | Luguru                    |
+| South America | Makonde                   |
+| South America | Nyakusa                   |
+| South America | Nyamwesi                  |
+| South America | Shambala                  |
+| South America | Acholi                    |
+| South America | Ganda                     |
+| South America | Gisu                      |
+| South America | Kiga                      |
+| South America | Lango                     |
+| South America | Lugbara                   |
+| South America | Nkole                     |
+| South America | Soga                      |
+| South America | Teso                      |
+| South America | Tagalog                   |
+| South America | Karakalpak                |
+| South America | Goajiro                   |
+| South America | Warrau                    |
+| South America | Man                       |
+| South America | Muong                     |
+| South America | Nung                      |
+| South America | Tho                       |
+| South America | Bislama                   |
+| South America | Futuna                    |
+| South America | Wallis                    |
+| South America | Samoan-English            |
+| South America | Soqutri                   |
+| South America | Northsotho                |
+| South America | Southsotho                |
+| South America | Venda                     |
+| South America | Xhosa                     |
+| South America | Bemba                     |
+| South America | Chewa                     |
+| South America | Lozi                      |
+| South America | Nsenga                    |
++---------------+---------------------------+
+457 rows in set (0,02 sec)
+
+mysql> 
+
+
+
+9 
+
+
+mysql> select city.Name , city.CountryCode,countrylanguage.Language  from city INNER  JOIN  countrylanguage ON countrylanguage.CountryCode = city.CountryCode where city.Name = 'Barcelona' ;
```

Comment:
Yes, this is technically correct, since both tables contain the same shared value from `country` - they both use `CountryCode`. However, if they were both related to `country` but used different values to relate to `country`, this wouldn't work - you would have to do a three-table join.